src/Components/NotFoundPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const NotFoundPage = (props) => {
+  return (
+    <div>
+      <h2>404 Page not found</h2>
+      <p>Hey looks like we couldn't find the page you were looking for!</p>
+      <p>Please use the menu/nav to find what you need?!?</p>
```

Comment:
We're not using props so we don't need to include it here.

Let's ask design for their input on what this page should look like.